File: AndroiAuthNative/AndroidAuthNativeTests/src/com/xamarin/auth/testutils/TestTrace.java
//
// Translated by CS2J (http://www.cs2j.com): 02/03/2014 00:30:30
//

package com.xamarin.auth.testutils;


import android.util.Log;

public class TestTrace
{
    public static void trace(String testName, TestStage stage)  {
        Log.e("MvxTraceLevel.Diagnostic", testName.concat(": ").concat(stage.toString()));
        //MvxTrace.Trace(MvxTraceLevel.Diagnostic, String.Format("{0}: {1}", testName, stage.ToString()));
    }

}

File: AndroidAuth.MvvmCross/Xamarin.Auth.WindowsPhone.Tests/TestTrace.cs
#if !UNMOD
using Cirrious.CrossCore.Platform;
#endif


namespace KeyStoreBug.DemoApp.Tests
{
    public enum TestStage { Arrange, Act, Assert }
    
    public static class TestTrace
    {
        public static void Trace(string testName, TestStage stage)
        {
#if UNMOD
            Android.Util.Log.Debug("MvxTraceLevel.Diagnostic", string.Format("{0}: {1}", testName, stage.ToString()));
#else
            MvxTrace.Trace(MvxTraceLevel.Diagnostic, string.Format("{0}: {1}", testName, stage.ToString()));
#endif
        } 
    }
}
